test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,37 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        //empty set
+        assertNotNull();
+
+        //some non-null objects
+        assertNotNull (1,2,3);
+        assertNotNull("");
+        assertNotNull("abc");
+        assertNotNull(new int[0]);
+        assertNotNull(new ArrayList());
+
+        //some null objects
+        assertIsNull (1,null);
+        assertIsNull("",null);
+        assertIsNull(null, 1, Integer.valueOf(1));
+
+        //all null objects
+        assertIsNull(null,null);
+        assertIsNull((Object)null);
+        String name = null;
+        assertIsNull (name);
+
+    }
+
+    private void assertNotNull (Object... objects){
+        assertFalse(Utils.isAnyNull(objects));
```

Comment:
Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.